eta/core/types.py

```diff
@@ -402,7 +402,7 @@ def gen_path(basedir, params):
 
     @staticmethod
     def is_valid_path(path):
-        return File.is_valid_path(path) and etav.is_supported_video(path)
+        return File.is_valid_path(path) and etav.is_supported_video_file(path)
```

Comment:
See comment above. Same.

Reply:
In our type system, `eta.core.types.Video` is an abstract type that code be an encoded video file `eta.core.types.VideoFile` or a sequence of images `eta.core.types.ImageSequence`.

`eta.core.video.SUPPORTED_VIDEO_FORMATS` only supports "video files" as defined above, so I updated the method names here to reflect that the related validation functionality is not handling image sequences.

In fact, I am also now renaming `SUPPORTED_VIDEO_FORMATS` to `SUPPORTED_VIDEO_FILE_FORMATS`, which should have also been done to drive home this distinction.